portfolio/src/main/webapp/index.html

```diff
@@ -9,114 +9,116 @@
+          <h3>February 2006 - May 2012 - Lodi, CA</h1>
+          <img src="../images/lakewood.jpg" alt="Lakewood Picture" class="img-60">
+          <p>
+            This is where I went to Elementary School. I spent 2006 - 2011 here. As mentioned previously, I started school at 1st grade
+            so the majority of my time here was spent catching up. I'll take the next few slides to introduce my town/schools.
+            Shoutout to <br> Mr. McKilligan, you're a real one.
+          </p>
+        </div>
+        <div class="mySlides fade">
+          <h1>Millswood Middle School</h1>
+          <h3>August 2012 - May 2014 - Lodi, CA</h1>
+          <img src="../images/millswood.jpg" alt="Millswood Picture" class="img-80">
+          <p>This is where I went to middle school. I don't remember much from this time period so we'll just move on.</p>
+        </div>
+        <div class="mySlides fade">
+          <h1>Lodi High School</h1>
+          <h3>August 2014 - May 2018 - Lodi, CA</h1>
+          <img src="../images/lodihigh.jpg" alt="Lodi High Picture" class="img-60">
+          <p>
+            This is my high school. I would say about 80% of my memories come from this place. I made a lot of freinds here through 
+            several activities such as cross country, track and field, rocketry club, the 3D printing lab, and Science Olympiad. As 
+            much as I hated this place while I was there, I miss the memories I made with my friends before we all went our respective paths.
+          </p>
+        </div>
+        <div class="mySlides fade">
+          <h1>High School Graduation</h1>
+          <h3>May 31st 2018 - Stockton, CA</h1>
+          <img src="../images/grad.jpg" alt="Grad Picture" class="img-75">
+          <p>These are some of my closest friends in high school. It was a very nostalgic day. I miss the boys.</p>
+        </div>
+        <div class="mySlides fade">
+          <h1>First Day of College</h1>
+          <h3>Septemeber 21st 2018 - Stanford, CA</h1>
+          <img src="../images/stanford.jpg" alt="Stanford Picture" class="img-75">
+          <p>
+            I started school on Septemeber 21st 2018. Going into college I was premed. After taking <b><a href="https://web.stanford.edu/class/cs106b/" target="_blank" id="cs-link">CS106B</a></b> (Programing Abstractions), I found the content very enagaging 
+            and decided to switch majors to Computer Science. College was very intimiating at first with everyone saying it would be extremely 
+            diffcult. Looking back to my first day, I would tell myself to take it easy.
+          </p>
+        </div>
+        <div class="mySlides fade">
+          <h1>First Day at Google</h1>
+          <h3>June 15th 2020 - Lodi, CA</h1>
+          <img src="../images/google.jpg" alt="Google Picture" class="img-50">
+          <p>I'm super exicted to be a part of Google and for all the explorations I take this summer!</p>
```

Comment:
Can you rephrase: for all the explorations I take this summer? Not sure what you mean.